test/heaped_vector.jl

```diff
@@ -0,0 +1,37 @@
+using IntervalOptimisation
+using Test
+
+const heap =IntervalOptimisation.heap
```

Comment:
Take care with punctuation: `=` should always have spaces on both sides.
Please check all your code for this.

Reply:
ok sure I'll edit the file.